src/js/utils/ParserUtil.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  combineParsers(parsers = []) {
+    parsers = parsers.filter(
+      function (parser) {
+        return typeof parser === 'function';
```

Comment:
Why this test here? Isn't the user of this function always supposing to pass functions as arguments?